cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -256,3 +258,64 @@ def document_type(self, key, value):
             raise IgnoreKey("document_type")
         else:
             raise UnexpectedValue(subfield="a")
+
+
+@model.over("_created", "(^916__)|(^595__)|(^961__)", override=True)
+@out_strip
+def created(self, key, value):
+    """Translates created information to fields."""
+    _created_by = self.get("created_by", {})
+    if key == "916__":
+        if "s" in value:
+            _created_by.update(
+                {
+                    "type": mapping(
+                        ACQUISITION_METHOD,
+                        clean_val("s", value, str, default="migration"),
+                        raise_exception=True,
+                    )
+                }
+            )
+            self["created_by"] = _created_by
+
+            # used to compare dates, to find the oldest publication date
+            # when multiple dates present in the field
+            date = 999999
+            date_values = clean_val(
+                "w", value, int, regex_format=r"\d{6}$", multiple_values=True
+            )
+            if type(date_values) is list:
+                for x in date_values:
+                    if x < date:
+                        date = x
+            else:
+                date = date_values
+            if date:
+                year, week = str(date)[:4], str(date)[4:]
+                date = get_week_start(int(year), int(week))
+                return date.isoformat()
+    elif key == "595__":
+        try:
+            sub_a = clean_val(
+                "a", value, str, regex_format=r"[A-Z]{3}[0-9]{6}$"
+            )
+            if sub_a:
+                source = sub_a[:3]
+                self["source"] = source
+                year, month = int(sub_a[3:7]), int(sub_a[7:])
+                self["_created"] = datetime.date(year, month, 1).isoformat()
+                raise IgnoreKey("_created")
+        except UnexpectedValue as e:
+            e.subfield = "a"
+            self["internal_notes"] = internal_notes(self, key, value)
+            raise IgnoreKey("_created")
+    elif key == "961__":
+        date_value = clean_val("x", value, int, regex_format=r"\d{8}$")
+        if date_value:
+            year, week, day = str(date_value)[:4],\
+                              str(date_value)[4:6],\
+                              str(date_value)[6:8]
+            date = datetime.date(int(year), int(week), int(day))
+            return date.isoformat()
```

Comment:
This is the real change here. Overrided this rule to add the translation of the field `961__x` for the journals, the rest is the same as for the other records (check [base.py](https://github.com/CERNDocumentServer/cds-ils/blob/master/cds_ils/importer/providers/cds/rules/base.py#L72-L122))